tests/integration/docusaurus/rule_based_profiler/multi_batch_example.py

```diff
@@ -0,0 +1,141 @@
+from ruamel import yaml
```

Comment:
Please put this script in a directory strucuture that matches the end docusaurus .md file.